cypress/support/commands/action.js

```diff
@@ -65,8 +65,8 @@ Cypress.Commands.add('executeQuickAction', (actionName, modal = false, isDialogE
   }
```

Comment:
About executeQuickAction, can you remove this comment "If the second parameter is truthy, the default action will be executed." from index.d.ts file as it's not up to date. I should have removed it when I removed the second param.

Reply:
Done